app/upgrades/v7/upgrades.go

```diff
@@ -17,11 +19,18 @@ func CreateUpgradeHandler(
 	configurator module.Configurator,
 	auctionKeeper auctionkeeper.Keeper,
 	cellarfeesKeeper cellarfeeskeeper.Keeper,
+	icaHostKeeper icahostkeeper.Keeper,
 	pubsubKeeper pubsubkeeper.Keeper,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 		ctx.Logger().Info("v7 upgrade: entering handler")
 
+		// TODO(bolten): get a sanity check on this
+		// Now that we're on IBC V6, we can update the ICA host module to allow all message types rather than
+		// the list we specified in the v6 upgrade -- a default of HostEnabled: true and the string "*" for messages
+		icaParams := icahosttypes.DefaultParams()
+		icaHostKeeper.SetParams(ctx, icaParams)
```

Comment:
Looks correct

https://github.com/cosmos/ibc-go/blob/v6.2.1/modules/apps/27-interchain-accounts/host/types/keys.go#L15

Reply:
Yeah, I think my concern is more whether or not we should allow all messages or stick with the original list we had. Any risk you think?

Reply:
I think all messages are fine